tests/openapi/test_turboquant.py

```
@@ -0,0 +1,325 @@
+            "vectors": {
+                "size": VECTOR_SIZE,
+                "distance": "Cosine",
+            },
+            "quantization_config": {
+                "turbo": {}
+            },
+        }
+    )
+    assert response.ok
+
+    rng = random.Random(0)
```

Ruff (0.15.12)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)